src/annotator/anchoring/test/types-test.js

```diff
@@ -28,26 +28,11 @@ describe('annotator/anchoring/types', () => {
   });
 
   describe('RangeAnchor', () => {
-    let fakeSniff;
-    let fakeSerializedRange;
-    let fakeNormalize;
-    let fakeSerialize;
+    let container;
```

Comment:
The updated tests for `RangeAnchor` follow the pattern for the test of the tests in this module of not mocking the `TextRange` or `TextPosition` classes.